src/components/account/RegisterCard.tsx

```diff
@@ -67,56 +60,50 @@ export default function RegisterCard(props: Props): ReactElement {
 
   return (
     <Form {...layout} name="registerCard" onFinish={onFinish} onFinishFailed={onFinishFailed}>
-      <Tooltip title="Back">
+      <Tooltip className="back-button" title="Back">
         <Button shape="circle" onClick={props.onClickBack} icon={<ArrowLeftOutlined />} />
       </Tooltip>
-      <h1>Create an account</h1>
+
+      <h1 className="register-account-title">Create an account</h1>
       <p>Create an account to upload promotions and save deals you like.</p>
-      <Form.Item style={{ marginBottom: 10 }}>
-        <Form.Item
-          name="firstName"
-          rules={InputRules.firstName}
-          style={{ display: 'inline-block', width: 'calc(50% - 6px)', marginRight: 6 }}
-          hasFeedback={true}
-        >
-          <Input placeholder="First name" />
-        </Form.Item>
-        <Form.Item
-          name="lastName"
-          rules={InputRules.lastName}
-          style={{ display: 'inline-block', width: 'calc(50% - 6px)', marginLeft: 6 }}
-          hasFeedback={true}
-        >
-          <Input placeholder="Last name" />
-        </Form.Item>
-      </Form.Item>
+
+      <Row gutter={6} style={{ marginBottom: 5 }}>
+        <Col span={12}>
+          <Form.Item name="firstname" rules={InputRules.firstName} hasFeedback={true}>
+            <Input placeholder="First name" />
+          </Form.Item>
+        </Col>
+        <Col span={12}>
+          <Form.Item name="lastname" rules={InputRules.lastName} hasFeedback={true}>
+            <Input placeholder="Last name" />
+          </Form.Item>
```

Comment:
Currently broken, needs to be:

```suggestion
          <Form.Item name="firstName" rules={InputRules.firstName} hasFeedback={true}>
            <Input placeholder="First name" />
          </Form.Item>
        </Col>
        <Col span={12}>
          <Form.Item name="lastName" rules={InputRules.lastName} hasFeedback={true}>
            <Input placeholder="Last name" />
          </Form.Item>
```

Reply:
Ah good catch thanks!